### DISCUSSION TITLE
How do I make an initial layout item full height?

### DISCUSSION THREAD

**Discussion by ninjaKris** [react-grid-layout/react-grid-layout#1713] [Q&A]
I'm setting up my layout as so
``` 
const layout = [
    { i: "a", x: 0, y: 0, w: 1, h: 6 },
    { i: "b", x: 2, y: 0, w: 5, h: 6 }
  ]
```
The columns are full height on my laptop monitor, but not for my larger monitor.

It would be nice to be able to set up my layout like so:
```
const layout = [
    { i: "a", x: 0, y: 0, w: 1, h: '100vh },
    { i: "b", x: 2, y: 0, w: 5, h: 100vh }
  ]
```

Is there anyway to make a column the height of the screen (100vh)?

Here is my GridLayout:
```
<GridLayout
        layout={layout}
        cols={6}
        width={window.innerWidth}
        autoSize={true}
      >
```

**Comment by CodyBontecou** [comment]
Bumping this. I'm curious as well.

**Reply by CodyBontecou** [reply]
Hmm, I found [this comment](https://github.com/react-grid-layout/react-grid-layout/issues/1715#issuecomment-1229244277) recommending [this package](https://github.com/jaredLunde/react-hook/tree/master/packages/window-size) to calculate the width and height and passing it to the component.
